tasks/local-registry.sh

```diff
@@ -29,5 +29,5 @@ function stopLocalRegistry {
 
 function publishToLocalRegistry {
   git clean -df
-  ./tasks/publish.sh prerelease --yes --force-publish=* --no-git-tag-version --no-commit-hooks --no-push --exact --dist-tag=latest
```

Comment:
Removed the prerelease due to an error from lerna that this command does not exist